lib/fluent/event.rb

```diff
@@ -21,6 +21,10 @@ class EventStream
     include Enumerable
     include MessagePackFactory::Mixin
 
+    def records
```

Comment:
Once I would use `length`, but I re-found that `Chunk#records` already exists, and I made `EventStream#records` as same name with it.
`Chunk` also has `Chunk#size` (bytesize), and `#size != #length` is extraordinary confusing in ruby way. Hmm.